src/app/stores/UserStore.ts

```diff
@@ -12,7 +12,7 @@ export class UserStore {
 
   constructor(rootStore: RootStore) {
     this.rootStore = rootStore
-    this.currentUser = null  // could be initialized from localStorage later
+    this.currentUser = ApiService.auth.getLocalStorageUser()
```

Comment:
arba galetumem visa sita logika imesti i onAuthStateChange, kad vietoj pirminio null userio duotu useri is persistance